hash_practice/exercises.py

```diff
@@ -2,28 +2,114 @@
 def grouped_anagrams(strings):

```

Comment:
👍 This works and you're right on the time complexity, space complexity seems to be O(n) because you're making a list of another list of words.

Could you use a dictionary to group the anagrams together making the key a tuple of all the letters, sorted and the value a list of words which are anagrams of these letters?